PoshRedisViewer/PoshRedisViewer/UILogic.fs

```diff
@@ -168,10 +168,24 @@ let setupViewsLogic multiplexer (views: Views) =
            |> ignore
        )
 
+
     views.ResultsListView
     |> ListView.addValueCopyOnRightClick id
     |> ListView.addValueCopyOnCopyHotKey id
-    |> ignore
+    |> fun resultsListView ->
```

Comment:
I think this can be extracted to a function as it doesn't seeem to capture any outer state. Like `addValueCopyOnCopyHotKey` above. Something along the lines of `addDetailedViewOnEnterKey`

Reply:
Done in [7e82db8](https://github.com/En3Tho/PoshRedisViewer/pull/11/commits/7e82db852aaf29386994507d474546601db64958)